client/src/modules/dashboard/Chapters/Users/pages/ChapterUsersPage.tsx

```diff
@@ -215,24 +215,25 @@ export const ChapterUsersPage: NextPage = () => {
                     {data.chapter ? (
                       <DataTable
                         data={[data.chapter.chapter_users[index]]}
-                        keys={['type', 'values'] as const}
+                        keys={['type', 'action'] as const}
+                        showHeader={false}
                         tableProps={{
                           table: { 'aria-labelledby': 'page-heading' },
                         }}
                         mapper={{
                           type: () => (
                             <VStack
                               spacing={3}
-                              fontWeight={500}
                               align={'flex-start'}
+                              marginBlock={'1em'}
                             >
-                              <Text>Name</Text>
-                              <Text>Email</Text>
-                              <Text>Ops</Text>
-                              <Text>Role</Text>
+                              <Text fontWeight={700}>Name</Text>
+                              <Text fontWeight={700}>Email</Text>
+                              <Text fontWeight={700}>Ops</Text>
```

Comment:
```suggestion
                              <Text fontWeight={700}>Actions</Text>
```
To be consistent with the rest.